### ISSUE TITLE
Jaws of Recovery do not fit in EMT belts or Medical belts

### ISSUE THREAD

**Issue by NomNumNomYum** [DopplerShift13/DopplerShift#1210]
### Client Version:

516.1663=>[516](https://www.byond.com/download/build/516)/[Windows](https://www.byond.com/download/build/516/516.1663_byond_setup.zip)/[Linux](https://www.byond.com/download/build/516/516.1663_byond_linux.zip)

### Issue Summary:

In the code, it says that the paramedics jaws of recovery should be able to fit into them, but they can't be and I couldn't figure out why this is the case personally

https://github.com/DopplerShift13/DopplerShift/blob/9712ccfae2fb42a359b8c74c122941767f1ad514/code/datums/storage/subtypes/belts.dm#L82

### Round ID:

[5791](https://statbus.space/round/5791)

### Test Merge Information:

- [[TESTMERGE ONLY] Temporary Fix to USB Cable Endless Recursion Hell](https://github.com/DopplerShift13/DopplerShift/pull/778)
- [Partially reverts #896, leaving donation boxes and supply shortages](https://github.com/DopplerShift13/DopplerShift/pull/968)
- [[TM ONLY] makes lizard eyes blink normally until we get the fix for them](https://github.com/DopplerShift13/DopplerShift/pull/1002)
- [[TM ONLY] Early pull of tgstation#95032: Honorifics don't mangle runechat color](https://github.com/DopplerShift13/DopplerShift/pull/1009)
- [Updates the character preview box](https://github.com/DopplerShift13/DopplerShift/pull/1013)
- [Adds Backstage OOC chat](https://github.com/DopplerShift13/DopplerShift/pull/1033)
- [Adds Telekinetic Punt to Powers & makes vending-machine-fu less opressive.](https://github.com/DopplerShift13/DopplerShift/pull/1140)
- [Makes miniature reality anchors dreadful to be around, turns them into machines with power draw, updates Manipulate to support unique interactions with objects, updates helpers with magic-flag checks](https://github.com/DopplerShift13/DopplerShift/pull/1160)
- [Summonable Bugfixes + QoL](https://github.com/DopplerShift13/DopplerShift/pull/1141)
- [[WORK IN PROGRESS] security resprite](https://github.com/DopplerShift13/DopplerShift/pull/1181)
- [[TM ONLY] (still missing the actual tether part) adds a handheld kinesis/tether tool to make shipbreaking not practically dependent on modsuits (and borg accessible)](https://github.com/DopplerShift13/DopplerShift/pull/1201)
- [updates and fixes mounted guns, makes the example weapon more usable](https://github.com/DopplerShift13/DopplerShift/pull/1198)

### Reproduction Steps:

1. Grab Jaws of Recovery
2. Grab a Medical belt
3. Try to put the Jaws in the belt
